components/contribute-cards/ContributeTier.js

```diff
@@ -157,13 +149,7 @@ const ContributeTier = ({ intl, collective, tier, ...props }) => {
                 <StyledLink
                   as={Link}
                   whiteSpace="nowrap"
-                  route="tier"
-                  params={{
-                    collectiveSlug: collective.slug,
-                    verb: 'contribute',
-                    tierSlug: tier.slug,
-                    tierId: tier.id,
-                  }}
+                  href={{ pathname: `/${collective.slug}/contribute/${tier.slug}`, query: { tierId: tier.id } }}
```

Comment:
`tierId` should be part of the URL, this one doesn't work: https://opencollective-frontend-nw4onuvfg.vercel.app/captainfact_io/contribute/tshirt?tierId=10355